docker-run.sh

```diff
@@ -231,10 +231,26 @@ run
 set -e
 
 if [[ ${DOLI_CRON} -eq 1 ]]; then
-    echo "PATH=\$PATH:/usr/local/bin" > /etc/cron.d/dolibarr
-    echo "*/5 * * * * www-data /var/www/scripts/cron/cron_run_jobs.php ${DOLI_CRON_KEY} ${DOLI_CRON_USER} > /var/www/documents/cron_run_jobs.php.log 2>&1" >> /etc/cron.d/dolibarr
-    cron -f
-    exit 0
+  DOLI_CRON_FILE=/etc/cron.d/dolibarr
```

Comment:
I'm not sure if it's really necessary to create a variable for that.

Reply:
is to be cleaner !